src/main/java/delete_event_use_case/DeleteEventPresenter.java

```diff
@@ -0,0 +1,14 @@
+package delete_event_use_case;
+
+/**
+ * @author : Name
+ * @created: November 19th 2022
+ * @since : 2022-11-21, Mon.
+ **/
```

Comment:
Missing the title of the class as well as the author for documentation and please move the documentation above the package to maintain consistency throughout everyone's files.